openoffload/cpp/framework/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM grpcbuild:v1beta1
 USER root
 ENV PATH="/home/grpc/local/bin:${PATH}"
 
-RUN mkdir -p /home/grpc/local/tests/{src,include,ssl}
+RUN mkdir -p /home/grpc/local/tests/[src,include,ssl]
```

Comment:
should be `{}`

Reply:
Also need to be run with `bash -c`. Fixed in latest commit.